### DISCUSSION TITLE
No documentation for v10.1.9 ?

### DISCUSSION THREAD

**Discussion by borderie** [singerdmx/flutter-quill#2094] [General]
Hello,

I'm trying to use the v10.1.9 on my flutter project but the documentation is almost only for v9 so most things don't work, I have to read the files in the vendors to try to find out how to do it but it's extremely tedious
Isn't there an alternative please?

thanks!

**Comment by AtlasAutocode** [comment]
Documentation is really important and there are active discussions about this.

Unfortunately, it is hard to do documentation and often times contributors don't have the time or don't think it is needed for that one little tweak. Or some other excuse... 

We are all volunteers and making time to create documentation is hard when we have so much else to do.

You say most things don't work. There were not big changes between versions 9 and 10. Can you give some examples or where you are having a problem? Very often it is easy to provide a quick tip, and it would provide guidance on what documentation is needed to get new users up and running quickly. 

The simplest advice is to copy code from the example app in the main repository.

**Comment by EchoEllet** [comment]
Thank you for the report. As @AtlasAutocode mentioned, we haven't introduced breaking changes other than minor breaking changes such as #2106, which has been merged today.

We have moved the `controller` outside of the configuration class in #2078. However, it's not a breaking change yet. We have deprecated the old parameter and updated the relevant docs in README.md. ([Deprecation message](https://github.com/singerdmx/flutter-quill/pull/2078/files#diff-49ca9b0fdd0d380a06b34d5aed7674bbfb27fede500831b3e1279615a9edd06dR91)).

We will wait for your response to fix anything that's outdated.
